auth/publickey.go

```diff
@@ -1,29 +1,27 @@
-package token
+package auth
 
 import (
 	"context"
 	"crypto/rsa"
 	"encoding/json"
 	"net/http"
 
-	"github.com/fabric8-services/fabric8-tenant/auth"
+	"github.com/fabric8-services/fabric8-common/log"
 	authclient "github.com/fabric8-services/fabric8-tenant/auth/client"
-	"github.com/fabric8-services/fabric8-tenant/configuration"
-	"github.com/fabric8-services/fabric8-wit/log"
 	"github.com/fabric8-services/fabric8-wit/rest"
 	errs "github.com/pkg/errors"
-	jose "gopkg.in/square/go-jose.v2"
+	"gopkg.in/square/go-jose.v2"
 )
 
 // GetPublicKeys returns the known public keys used to sign tokens from the auth service
-func GetPublicKeys(ctx context.Context, authURL string, options ...configuration.HTTPClientOption) ([]*rsa.PublicKey, error) {
-	client, err := auth.NewClient(authURL, "", options...) // no need for a token when calling this endpoint
+func (s *Service) GetPublicKeys() ([]*rsa.PublicKey, error) {
```

Comment:
Any particular reason to keep this method in a separate file (from auth/service.go)?

Reply:
well, the `auth/service.go` file seemed to be big enough and originally it was in this file `auth/publickey.go` so I kept it there. But you are right that it would make more sense to keep all methods with the same receiver in the same file :+1: and there is no significant reason for keeping it separated.

Reply:
refactored in d84fbd8